semantic-scope-graph/src/Convert/ToScopeGraph.hs

```diff
@@ -0,0 +1,73 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE ExistentialQuantification #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE KindSignatures #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeOperators #-}
+module Convert.ToScopeGraph
+  ( ToScopeGraph (..)
+  , Result (..)
+  , onChildren
+  , onField
```

Comment:
📝 for the module and the stuff in it please?

Also, I would prefer if we kept with the habit of naming module hierarchies after the package: `semantic-tags` vends `Tags.*`, `semantic-analysis` vends `Analysis.*`, and preferably `semantic-scope-graph` should vend `ScopeGraph.*`. `ScopeGraph.Convert`?